src/documents/bulk_edit.py

```diff
@@ -321,31 +344,35 @@ def split(
     doc_ids: list[int],
     pages: list[list[int]],
     delete_originals: bool = False,
-    user: User = None,
-):
+    user: User | None = None,
+) -> Literal["OK"]:
     logger.info(
         f"Attempting to split document {doc_ids[0]} into {len(pages)} documents",
     )
     doc = Document.objects.get(id=doc_ids[0])
     import pikepdf
 
-    consume_tasks = []
+    consume_tasks: list = []
```

Comment:
Again with celery, this probably doesn't help type anything since the signature is not really used directly anyway